e2e_tests/tests/cluster/test_agent_restart.py

```diff
@@ -270,14 +270,14 @@ def test_agent_reconnect_keep_cmd(managed_cluster: ManagedCluster) -> None:
     managed_cluster.ensure_agent_ok()
 
     try:
-        command_id = run_command(30, slots=0)
```

Comment:
why did this get changed?

Reply:
Thank you for catching this, must have happened when I tried to revert `e2e_tests`